openmetadata-ui/src/main/resources/ui/src/components/Customization/GenericProvider/GenericProvider.tsx

```diff
@@ -151,30 +151,60 @@ export const GenericProvider = <T extends Omit<EntityReference, 'type'>>({
 
   // Handle the left side panel expand collapse
   useEffect(() => {
+    if (!leftPanelWidget) {
+      return;
+    }
+
```

Comment:
Instead of returning undefined from useEffect, can't we just perform the below in reverse condition? That'd be a cleaner way IMO